### ISSUE TITLE
Accessibility - Step Topics Screens - Missing Headings

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#38]
The different Steps "Topic" screens are missing heading which screen reader users use to navigate the screen.

**Comment by adriennerlong** [comment]
The developers are aware of this problem and are making it a high priority to address before 9/11.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Thanks, for getting the developers to fix this item. It's important for those using screen readers.

**Comment by jfc3-dol** [comment]
There is a heading level one (h1) on the page after the header sections and the bread crumbs but the other bold text should be a heading level two (h2) item. Those would be the four or five main content sections.

**Comment by jfc3-dol** [comment]
@adriennerlong,

The Topic screens now seem to have additional headings besides the top on e as a heading level one (h1).

On the Step 3 Topic 3 screen there are three or four heading level three (h3) items in a row with no other content in between them.

The heading level three (h3) items are the "Example in Action". "Responsible Sourcing Tool 1", Protections against Trafficking in Persons", and Sample Code of Conduct Provisions".

Plus, the light blue text on the gray background does not have enough color contrast.

Will look through more screens tomorrow (Friday) when I have more time because right now I'm attempting to close as many completed issues as I can today.

**Comment by adriennerlong** [comment]
From ICF: Fixed

**Comment by jfc3-dol** [comment]
From the random pages, I checked they all had a heading level one  (h1) and at least one heading level two (h2) if not more. A few topic screens even had heading level three (h3) text.

Removing the iOS label and will check the Android and web version when I have them available to me.

**Comment by jfc3-dol** [comment]
From the random pages, I checked they all had a heading level one (h1) and at least one heading level two (h2) if not more. A few topic screens even had heading level three (h3) text.

Removing the TalkBack label and will check the web version when I have them available to me.

**Comment by jfc3-dol** [comment]
From the random pages, I checked they all had a heading level one (h1) and at least one heading level two (h2) if not more. A few topic screens even had heading level three (h3) text.

Removing the web label and will close the issue.
